hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestContainerEndpoint.java

```diff
@@ -759,7 +760,7 @@ public void testGetMissingContainers() throws IOException, TimeoutException {
         new HashSet<>(Arrays.asList("host2", "host3", "host4")));
     List<ContainerHistory> containerReplicas = container.getReplicas();
     containerReplicas.forEach(history -> {
-      Assert.assertTrue(datanodes.contains(history.getDatanodeHost()));
+      Assertions.assertTrue(datanodes.contains(history.getDatanodeHost()));
```

Comment:
`assertTrue` is imported, can be used directly.

Reply:
Good catch. modified these changes.